config/default/capm3/namespace.yaml

```diff
@@ -2,3 +2,6 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: capm3-system
+  labels:
+    pod-security.kubernetes.io/enforce: restricted
```

Comment:
while this seems a best practice & recommended security hardening, I have not seen something similar in the CAPI/*providers repos. Is there any specific use case for having this?

Reply:
Where can we check the security constraints definition for the restricted value?

Reply:
https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted

Reply:
@furkatgofurov7 The use case is exactly that: it doesn't change anything per se, but it also reinforces security posture, by making it impossible to create pods that violate the security standard in the namespace.